qubes/tools/qubesd_query.py

```diff
@@ -35,6 +36,20 @@
 parser.add_argument('arg', metavar='ARGUMENT',
     nargs='?', default='',
     help='argument to method')
+parser.add_argument('--max-bytes',
+    dest='max_bytes',
+    type=int,
+    help='Maximum number of bytes to read from stdin')
+parser.add_argument('--single-line',
+    dest='single_line',
+    action='store_true',
+    default=False,
+    help='Only read a single line of input')
+parser.add_argument('--null',
+    dest='null',
+    action='store_true',
+    default=False,
+    help="Print an extra NUL byte; useful for shell scripts")
```

Comment:
Can you clarify what's the point of this? Not only the option, but also printing this byte at all. Preferably in man page's description of this option (which is missing, for other options added here too).

Reply:
I added documentation for this option.  The other new options have documentation already IIUC.